javascript/javascript3/week2/promise.js

```diff
@@ -0,0 +1,93 @@
+ // Promise that resolves after set time
+
+function getResolveIn(resolveAfter) {
+   return new Promise(function(resolve, reject){
+       setTimeout(function() {
+           resolve(resolveAfter);
```

Comment:
You should just write "resolve();" without any input.

Reply:
ok